pkg/cache/cache.go

```diff
@@ -135,7 +135,6 @@ type ClusterQueue struct {
 	Status    metrics.ClusterQueueStatus
 
 	// The following fields are not populated in a snapshot.
-
```

Comment:
Plz revert this, the comment is for all fields below, to avoid confusing with comment for `admittedWorkloadsPerQueue` only.